app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/SearchResults.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="flex justify-end gap-1 py-4 bg-white dark:bg-slate-900">
+    <div class="flex flex-col gap-1 w-full">
+      <div v-if="isLoading" class="empty-state-message">
+        <spinner size="" />
+        {{ $t('HELP_CENTER.ARTICLE_SEARCH_RESULT.SEARCH_LOADER') }}
+      </div>
+      <div
+        v-else-if="articles.length === 0 && searchQuery"
+        class="empty-state-message"
+      >
+        {{ $t('HELP_CENTER.ARTICLE_SEARCH_RESULT.NO_RESULT') }}
+      </div>
+      <search-result-item
+        v-for="article in articles"
+        v-else
+        :id="article.id"
+        :key="article.id"
+        :title="article.title"
+        :body="article.content"
+        :url="article.url"
+        :category="article.category.name"
+        :locale="article.category.locale"
+        @preview="handlePreview"
+        @insert="handleInsert"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import Spinner from 'shared/components/Spinner';
```

Comment:
@scmmishra Can we add a lint rule to use .vue everytime? Otherwise we would have to run the migration work we will did again.